charts/alfresco-process-services/values.yaml

```diff
@@ -146,9 +156,23 @@ adminApp:
     timeoutSeconds: 5
   readinessProbe: *adminProbe
   ingress:
-    className: "nginx"
-    path: /activiti-admin
-    maxUploadSize: "5G"
+    # -- Toggle ingress for APS application
+    enabled: true
+    # -- provide annotations for nginx ingress (no toher ingress is supported at
+    # that point)
+    annotations:
+      nginx.ingress.kubernetes.io/proxy-body-size: 5g
+      nginx.ingress.kubernetes.io/affinity: "cookie"
+    # -- supported ingress class
+    className: nginx
```

Comment:
check typo and not sure why we need to explicit what we support here, it's pretty self-explanatory and doesn't prevent users to provide their own config

Reply:
right that's actually a copy/paste from repo